src/eventHandlers/checkSuite/index.ts

```diff
@@ -124,6 +128,8 @@ const tryMerge = async (
     );
   } else if (pullRequestState !== 'OPEN') {
     logInfo(`Pull request is not open: ${pullRequestState}.`);
+  } else if (shouldMerge(pullRequestTitle) === false) {
+    logInfo(`Pull request version bump is not allowed by PRESET.`);
```

Comment:
I'm open to adjusting the wording of this log message or adding more relevant information. I noticed that all other reasons for skipping a merge had log messages associated with them and I tried to keep consistent with that.